src/components/CountrySelector/style.scss

```diff
@@ -53,3 +43,201 @@
+  position: absolute;
+  left: 12 * $unit;
+  top: 10 * $unit;
+  width: 24 * $unit;
+  height: 24 * $unit;
+  background-color: transparent;
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  border-radius: 50%;
+}
+
+.countryLabel {
+  padding-left: 35 * $unit;
+  display: flex;
+  align-items: center;
+}
+
+//-----------------------------//
+// accessible-autocomplete CSS //
+//-----------------------------//
+/*
+  All selector default BEM block name replaced with our custom CSS namespace
+  i.e. '.autocomplete__' to '.custom__'
+
+  NOTE (as per library's documentation):
+  "You will need to rewrite the CSS class names to use your specified block name."
+
+  Copied as is from node_modules/accessible-autocomplete/src/autocomplete.css
+  except for the following confirmed unused styles:
+    - media query for @media (min-width: 641px) styling
+    - .custom__option--odd
+
+  Changes only applied for linting errors and to apply SDK theme styles
+  Use Sass variables, placeholders as much as possible to easily maintain this
+*/
+
+$_sdk-font-size: $font-size-small;
+$_sdk-line-height: 24 * $unit-small;
+$_sdk-text-color: $color-body-text;
+
+$_sdk-border-style: 1px solid $color-input-border;
+$_sdk-input-horizontal-padding: 8 * $unit-small;
+$_sdk-input-vertical-padding: 12 * $unit-small;
+$_sdk-input-icon-padding: 34 * $unit-small;
+
+// $_sdk-option-hover-color: #3640f5;
+$_sdk-option-hover-color: $color-primary-button-hover;
+$_sdk-option-active-color: $color-primary-button-active;
+$_sdk-option-padding: 8 * $unit-small;
+$_sdk-option-border-bottom: 0;
+
+%_sdk-caret-icon {
+  background-color: $color-white;
+  background-image: url('assets/search-icon.svg');
+  background-size: 24 * $unit-small;
+  background-repeat: no-repeat;
+  background-position: 98% 50%;
+}
+
+.custom__wrapper {
+  position: relative;
+}
+
+.custom__hint,
+.custom__input {
+  height: 100%;
+  border: $_sdk-border-style;
+  border-radius: 4 * $unit;
+  box-sizing: border-box;
+  margin-bottom: 0; /* BUG: Safari 10 on macOS seems to add an implicit margin. */
+  width: 100%;
+}
+
+.custom__input {
+  background-color: transparent;
+  position: relative;
+}
+
+.custom__hint {
+  color: #b1b4b6;
+  position: absolute;
+}
+
+.custom__input--default {
+  @extend %_sdk-caret-icon;
+  padding: $_sdk-input-horizontal-padding $_sdk-input-icon-padding
+    $_sdk-input-horizontal-padding $_sdk-input-vertical-padding;
+}
+
+.custom__input--focused {
+  /* outline: 3px solid #fd0; Commented out as SDK style for this is currently to use browser default */
+  outline-offset: 0;
+  box-shadow: inset 0 0 0 2px;
+}
+
+.custom__input--show-all-values {
+  padding: 5px 34px 5px 5px; /* Space for arrow. Other padding should match .custom__input--default. */
+  cursor: pointer;
+}
+
+.custom__dropdown-arrow-down {
+  z-index: 0;
+  display: inline-block;
+  position: absolute;
+  right: 8px;
+  width: 24px;
+  height: 24px;
+  top: 10px;
```

Comment:
Why are we `px` here (and in other places within this file)?

Reply:
This is the library's default CSS that was copied over as-is. I could remove these and edit the styling for `.custom__dropdown-arrow-down` to `display: none;` since it isn't used here the SDK UI and is only displayed when `showAllValues` is enabled. Or I could remove the style block all together. Though that results in a displaced dropdown arrow element if `showAllValues` ever is enabled.

Reply:
If we don't need it then we should remove it. If we need it, we should make sure that using px still allows each div to shrink or expand when the font size is changed at browser level